app/test_main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from app.main import can_access_google_page
+
+from unittest import mock
+import pytest
```

Comment:
https://peps.python.org/pep-0008/#:~:text=Imports%20are%20always,library%20specific%20imports.
fix this

Reply:
Fixed